manual/opt.tex

```diff
@@ -251,6 +253,10 @@ \subsubsection{adaptive}
   \item \ixml{shift_s}.  This is the initial coefficient used to scale the overlap-based stabilizer.
                             More stable but slower optimization is expected with a large value.
                             The adaptive method will automatically adjust this value after each linear method iteration.
+  \item \ixml{target_shift}.  If set greater than zero, the adaptive method will choose the update whose shift_i value is closest to
```

Comment:
Update here accordingly for the name change.

Reply:
Manual updated accordingly.